specification/hybridcompute/resource-manager/readme.md

```diff
@@ -410,4 +410,8 @@ suppressions:
     from: HybridCompute.json
     where: $.definitions.AgentVersion
     reason: Previously approved and released, would require potentially breaking changes
+  - code: GetCollectionResponseSchema
+    from: HybridCompute.json
+    where: $.paths["/subscriptions/{subscriptionId}/providers/Microsoft.HybridCompute/locations/{location}/publishers/{publisher}/extensionTypes/{extensionType}/versions"]
+    reason: Response model of GET collection operation "ExtensionMetadataV2_List" should not match with the response model of individual GET operation "ExtensionMetadataV2_Get". The individual GET operation returns additional properties that we do not want to include in the response model of the collection operation.
```

Comment:
Are you sure it is worth violating the ARM guidelines, just to not return the properties in the collection?  Why is not returning the properties in the collection more important than consistency across APIs?

Reply:
I don't see the need to maintain this consistency between LIST and GET APIs. Our list API lists out all the versions, while GET API response provides more information about a particular version.

Reply:
It is part of the RPC contract:

https://eng.ms/docs/products/arm/api_contracts/guidelines/rpc#rpc008-put-get-patch--list-must-return-the-same-resource-schema

https://github.com/cloud-and-ai-microsoft/resource-provider-contract/blob/master/v1.0/get-resource.md#rules

Reply:
We discussed and decided to conform to ARM guidelines.